plugins/imagepath.js

```diff
@@ -0,0 +1,10 @@
+export default defineNuxtPlugin(() => {
+    return {
+        provide: {
+            imagePath: (path) => {
+                const runtimeConfig = useRuntimeConfig()
+                return `${runtimeConfig.app.baseURL}/images/${path}`
```

Comment:
僕の環境だと「//images/sponsor/serio.png」みたいにimagesの前に//がついてしまって上手くパス解決出来てないみたいです

```suggestion
                const pathlib = require('path')
                return pathlib.join(runtimeConfig.app.baseURL, 'images', path)
```

のようにpath結合するためメソッド呼んだ方が良さそうな気がするんですがどうでしょう？